learn/core_concepts/documents.md

```diff
@@ -1,26 +1,58 @@
 # Documents
 
-A **document** is an object composed of one or more **fields**. Each field consists of an **attribute** and its associated **value**.
-
-Documents function as **containers for organizing data**, and are the basic building blocks of a Meilisearch database. To search for a document, it must first be added to an [index][indexes].
+A document is an object composed of one or more fields. Each field consists of an **attribute** and its associated **value**. Documents function as containers for organizing data, and are the basic building blocks of a Meilisearch database. To search for a document, it must first be added to an [index](/learn/core_concepts/indexes.md).
 
 ## Structure
 
 ![Diagram illustration Meilisearch's document structure](/document_structure.svg =573x400)
 
 ### Important terms
 
-- **Document**: an object which contains data in the form of one or more fields
-- **[Field][fields]**: a set of two data items that are linked together: an **attribute** and a **value**
-- **Attribute**: the first part of a field. Acts as a name or description for its associated value.
-- **Value**: the second part of a field, consisting of data of any valid JSON type
-- **[Primary Field][primary-field]**: A special field that is mandatory in all documents. It contains the primary key and document identifier.
-- **[Primary Key][primary-key]**: the attribute of the primary field. **All documents in the same index must possess the same primary key.** Its associated value is the document identifier.
-- **[Document Identifier][document-id]**: the value of the primary field. **Every document in a given index must have a unique identifier**.
+- **Document**: An object which contains data in the form of one or more fields
+- **[Field](#fields)**: A set of two data items that are linked together: an attribute and a value
+- **Attribute**: The first part of a field. Acts as a name or description for its associated value
+- **Value**: The second part of a field, consisting of data of any valid JSON type
+- **[Primary Field](#primary-field)**: A special field that is mandatory in all documents. It contains the primary key and document identifier
```

Comment:
I thought we had agreed on capitalizing the first letter after a colon only when it was used in headings? Not sure I'm sold on capitalizing it on list items.